### ISSUE TITLE
agent reply to new internal activity email alerts will set all ticket thread in yellow.

### ISSUE THREAD

**Issue by sjxiong** [osTicket/osTicket#5565]
### Prerequisites

* [ yes ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [ no ] Do you have any errors in the PHP error log, or javascript console?
* [ yes ] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [ yes ] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

agent reply to new internal activity email alerts will set all ticket thread in yellow.

[Description of the bug or feature]

### Steps to Reproduce

1. an Agent post an internal note in a ticket.
2. The assigned agent has received an internal activity alert email.
3. The assigned agent replied directly the alert email with a comment.
4. The comments is registered in the ticket as internal note. And it set all ticket threads in yellow.

**Expected behavior:** The agent reply should be registered as internal note. (in fact, he should not direct email reply to an alert email, but some people do. and we have tested, the replies will also be added to the ticket, but by doing this, all threads in the ticket will be turned to yellow

**Actual behavior:** The reply is added to the ticket as internal note, but it sets all the threads in yellow.
![image](https://user-images.githubusercontent.com/15185770/85704400-8ba50a00-b6e0-11ea-88c8-62dfe19aff7d.png)

### Versions

Server Information
--
osTicket Version | v1.14.1 (f1e9e88)  Upgrade— v1.14.2 is available
Web Server Software | Apache/2
MySQL Version | 10.3.23
PHP Version | 7.3.19

Windows10

**Comment by JediKev** [comment]
@sjxiong 

I would suggest upgrading to the latest release of v1.14.3 and retest. If any Agent replies to any alert via email it will be added to the ticket as an Internal Note. However, this should not affect previous thread entries or new thread entries.

Cheers.

**Comment by sjxiong** [comment]
We have now v1.15, and still, it will affect the previous thread. have you done a test? I noticed that this only happens with the agent view, if I log in as a user, all threads are shown correctly.

**Comment by JediKev** [comment]
@sjxiong 

Yes, in my testing replying as an Agent to an email will add as Internal Note and not affect anything else. Maybe the content in the email is being cutoff and the now broken HTML is affecting the page's HTML? Look for the thread entry in the database and see if the HTML is complete.

Cheers.
